tests/test_application.py

```diff
@@ -267,3 +268,31 @@ def helper(scopes, account, authority, *args, **kwargs):
         with self.assertRaises(ExpectedBehavior):
             self.app.acquire_token_silent(["different scope"], self.account)
 
+
+class TestApplicationForClientCapabilities(unittest.TestCase):
+
+    def test_merged_claims_returned_correctly(self):
```

Comment:
I believe our org encourages a more informational test method naming pattern "condition_a_b_c_should_result(...)" e.g. "test_divide_by_zero_should_raise_exception()". We might not always enforce that, but the current name is just too vague to differentiate it from its cohort. :-)
```suggestion
    def test_capability_and_id_token_claims(self):
```